src/stable_browser.ts

```
@@ -2486,13 +2492,15 @@ class StableBrowser {
+      const resultWithElementsFound = {
+        length: null, // initial cannot be 0
+      };
       while (true) {
-        const resultWithElementsFound = await this.findTextInAllFrames(
-          dateAlternatives,
-          numberAlternatives,
-          text,
-          state
-        );
+        try {
+          resultWithElementsFound = await this.findTextInAllFrames(dateAlternatives, numberAlternatives, text, state);
```

Biome (1.9.4)

[error] Can't assign resultWithElementsFound because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)